src/scripts/fetch_user_handles.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 set -eo pipefail
 USER_EMAIL=""
-CIRCLE_USERNAME=$CIRCLE_USERNAME
 function run_main() {
     USER_ALIAS=$(curl -s -H "Authorization: Bearer ${CODA_API_TOKEN}" \
     -G --data-urlencode "query=${CODA_CIRCLECI_USER_NAME_COL}:\"${CIRCLE_USERNAME}\"" \
     "${CODA_USER_ROSTER_TABLE_URL}" \
     | jq --arg CODA_CIRCLECI_USER_ALIAS_COL "$CODA_CIRCLECI_USER_ALIAS_COL" '.items[0].values."'"$CODA_CIRCLECI_USER_ALIAS_COL"'"' | tr -d '"')
+
     if [ "$USER_ALIAS" != "null" ]; then
         USER_EMAIL=$([[ "${USER_ALIAS}" == *@* ]] && echo "$USER_ALIAS" || echo "${USER_ALIAS}@${EMAIL_DOMAIN}")
         echo "$USER_EMAIL"
         echo "export USER_EMAIL=${USER_EMAIL}" >> "$BASH_ENV"
-        
+    else
+        echo "export USER_EMAIL=false" >> "$BASH_ENV"
```

Comment:
Why `false` and not empty string?

Reply:
I could export empty string, I just have to export something if variable was not set it would still go into the condition